charts/kubescape-operator/templates/node-agent/daemonset.yaml

```diff
@@ -53,6 +53,19 @@ spec:
       serviceAccountName: {{ .Values.nodeAgent.name }}
       automountServiceAccountToken: true
       hostPID: true
+      {{- if .Values.nodeAgent.initContainer }}
+      initContainers:
+      - name: init-delay
+        image: "busybox:latest"
+        command: 
+        - /bin/sh
+        - -c
+        - |
+          SLEEP_TIME=$(( $RANDOM % 120 ))
```

Comment:
Make max number of seconds configurable by the chart's values

Reply:
Done